vnext/Scripts/rnw-dependencies.ps1

```diff
@@ -65,18 +65,13 @@ $vsComponents = @('Microsoft.Component.MSBuild',
     'Microsoft.VisualStudio.Component.VC.Tools.x86.x64',
     'Microsoft.VisualStudio.ComponentGroup.UWP.Support',
     'Microsoft.VisualStudio.ComponentGroup.NativeDesktop.Core',
-    'Microsoft.VisualStudio.Component.Windows10SDK.19041');
+    'Microsoft.VisualStudio.Component.Windows11SDK.22621');
```

Comment:
Can you leave in 19041 for now? This setup script is still used by 0.74 users, and they still need 19041. Eventually we'll need to update the documentation to reflect that different versions need to run different scripts, but in the meantime, leaving this in just means some wasted space.

```suggestion
    'Microsoft.VisualStudio.Component.Windows10SDK.19041',
    'Microsoft.VisualStudio.Component.Windows11SDK.22621');
```